docs/Index.tsx

```diff
@@ -1,78 +1,82 @@
-import { ModalExamples } from '../src/components/modal/examples/ModalExamples';
-import { ModalPromptExamples } from '../src/components/modalPrompt/exemples/ModalPromptExamples';
-import { MultilineInputExamples } from '../src/components/multilineInput/examples/MultilineInputExamples';
-import { NavigationConnectedExamples } from '../src/components/navigation/examples/NavigationConnectedExamples';
-import { NavigationExamples } from '../src/components/navigation/examples/NavigationExamples';
-import { OptionsCycleConnectedExamples } from '../src/components/optionsCycle/examples/OptionsCycleConnectedExamples';
-import { OptionsCycleExamples } from '../src/components/optionsCycle/examples/OptionsCycleExamples';
-import { RadioExamples } from '../src/components/radio/examples/RadioExamples';
-import { SideNavigationLoadingExample } from '../src/components/sideNavigation/examples/SideNavigationLoadingExample';
-import { SliderExamples } from '../src/components/slider/examples/SliderExamples';
-import { SplitLayoutExamples } from '../src/components/splitlayout/examples/SplitLayoutExamples';
-import { StepProgressBarExamples } from '../src/components/stepProgressBar/examples/StepProgressBarExamples';
-import { SubNavigationConnectedExamples } from '../src/components/subNavigation/examples/SubNavigationConnectedExamples';
-import { SubNavigationExamples } from '../src/components/subNavigation/examples/SubNavigationExamples';
-import { LinkSvgExamples } from '../src/components/svg/examples/LinkSvgExamples';
-import { SvgExamples } from '../src/components/svg/examples/SvgExamples';
-import { SyncFeedbackExample } from '../src/components/syncFeedback/examples/SyncFeedbackExample';
-import { TabsExamples } from '../src/components/tab/examples/TabConnectedExample';
-import { TableWithDisabledRowsExamples } from '../src/components/tables/examples/TableDisabledRowsExamples';
-import { TableEmptyRowExamples } from '../src/components/tables/examples/TableEmptyRowExamples';
-import { TableExamples } from '../src/components/tables/examples/TableExamples';
-import { TableHeaderExamples } from '../src/components/tables/examples/TableHeaderExamples';
-import { TableRowConnectedExamples } from '../src/components/tables/examples/TableRowConnectedExamples';
-import { TableRowExamples } from '../src/components/tables/examples/TableRowExamples';
-import { TitleExamples } from '../src/components/title/examples/TitleExamples';
-import { ToastConnectedExamples } from '../src/components/toast/examples/ToastConnectedExamples';
-import { ToastExamples } from '../src/components/toast/examples/ToastExamples';
-import { TooltipExamples } from '../src/components/tooltip/examples/TooltipExamples';
-import { UserFeedbackExample } from '../src/components/userFeedback/examples/UserFeedbackExample';
-import { MembersExample } from './members-example/MembersExample';
-import { ReactVaporStore } from './ReactVaporStore';
+import {render as ReactDOMRender} from 'react-dom';
```

Comment:
Did we change the lintfix for this? IIRC everywhere else in react-vapor there is a space before and after the import brackets.

Reply:
pretty-typescript used to do this for us. lintfix does not and if it did, it would remove the spaces as well. To get it back to what it used to be, I would have to do it by hand. I do not mind doing it, but I thought that since this is what's in our code style, I would leave it as is so it would slowly become standard in the repo.